docs/docs-content/clusters/palette-virtual-clusters/configure-oidc-virtual-cluster.md

```diff
@@ -0,0 +1,106 @@
+
+5. In the **Host Clusters** tab that displays on the next page, click on **Settings**.
+
+6. In the slide panel that opens, select the **Settings** option. The cluster group YAML file displays.
+
+7. Locate the `vcluster.extraArgs` parameter section of the cluster group configuration file and uncomment the lines shown in the example.
+
+![Screenshot of the cluster group YAML showing oidc-related parameters to uncomment and update.](/clusters_palette-virtual-clusters_configure-vcluster-oidc.png)
+
+8. Update the `vcluster.extraArgs` section with your identity provider information as shown in example.
+
+    ```yaml
+    vcluster:
+      extraArgs:	
+      - --kube-apiserver-arg=”oidc-issuer-url=<client-url>"
+      - --kube-apiserver-arg="oidc-client-id=<client-id>"
+      - --kube-apiserver-arg="oidc-username-claim=email"
+      - --kube-apiserver-arg="oidc-groups-claim=groups"
+    ```
+
+9. If your identity provider requires a client secret, uncomment the `oidc-client-secret` parameter in the `clientConfig` section of cluster group YAML file, and add the client secret.
+
+    ```yaml
+    clientConfig:	
+      oidc-client-secret: secret-value
+    ```
+
+10. From the **User Menu**, switch to *App Mode* and click on **Virtual Clusters**.  
+
+11. Deploy a virtual cluster to the cluster group that you configured with OIDC. For steps, review the [Deploy a Virtual Cluster](../palette-virtual-clusters/deploy-virtual-cluster.md#deploy-a-virtual-cluster) guide. 
+
+  :::info
+
+  If the cluster group is part of a project, you can deploy a virtual cluster in *Cluster Mode* from **Cluster Groups** > **Virtual Clusters**.
+
+  :::
+
+  When the virtual cluster is finished deploying and in **Running** state, a Kubeconfig file is generated that contains OIDC information inserted into it from the cluster group configuration.
+
+12. Use the **Kubeconfig** link that displays on the virtual clusters overview page to download the Kubeconfig file. This will give you access to the Kubernetes cluster.
+
+  In *App Mode* click on **Virtual Clusters** and select the virtual cluster. In *Cluster Mode* navigate to **Cluster Groups** > **Virtual Clusters** and select the virtual cluster.
+
+13. Configure Role-Based Access Control (RBAC) for the virtual cluster. 
+
+:::caution
+
+Configuring OIDC requires you to map a set of users or groups to a Kubernetes RBAC role. To learn how to map a Kubernetes role to users and groups, refer to [Create Role Bindings](../cluster-management/cluster-rbac.md#create-role-bindings). Refer to [Use RBAC with OIDC](../../integrations/kubernetes.md#use-rbac-with-oidc) for an example. 
+
+:::
+
+
+## Validate
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. Navigate to your virtual clusters, either from **Cluster Groups** > **Virtual Clusters** in *Cluster Mode* if the cluster group is part of a project, or by switching to *App Mode*  from the **User Menu** and selecting **Virtual Clusters**.
```

Comment:
We can keep both modes, but let's improve the navigation. The `>` is not helpful, in my opinion, from a beginner's perspective.

Reply:
Spelled out navigation.